packages/adapter/addon/rest.js

```diff
@@ -1301,11 +1301,13 @@ function fetchSuccessHandler(adapter, payload, response, requestData) {
 
 function fetchErrorHandler(adapter, payload, response, errorThrown, requestData) {
   let responseData = fetchResponseData(response);
+
```

Comment:
@aclarembeau total nitpick, but can you remove this whitespace?

Reply:
Sure, it's done :)